### ISSUE TITLE
About the learning rate

### ISSUE THREAD

**Issue by smandava98** [shubham-goel/4D-Humans#65]
Hi, 

I'm struggling to understand a particular piece of code regarding the optimizers here:

```
def configure_optimizers(self) -> Tuple[torch.optim.Optimizer, torch.optim.Optimizer]:
        """
        Setup model and distriminator Optimizers
        Returns:
            Tuple[torch.optim.Optimizer, torch.optim.Optimizer]: Model and discriminator optimizers
        """
        param_groups = [{'params': filter(lambda p: p.requires_grad, self.get_parameters()), 'lr': self.cfg.TRAIN.LR}]

        optimizer = torch.optim.AdamW(params=param_groups,
                                        # lr=self.cfg.TRAIN.LR,
                                        weight_decay=self.cfg.TRAIN.WEIGHT_DECAY)
```

May I ask why a learning rate warmup scheduler was not used?

I've read that training a from scratch transformer is quite difficult without some warmup so just curious.

Thanks!

**Comment by smandava98** [comment]
I see there is no need with prenorm:

https://arxiv.org/abs/1910.05895
